deploy.py

```diff
@@ -1,142 +1,134 @@
 #!/usr/bin/env python3
 
-# Copyright (C) 2019 Adrian Carpenter
+# Copyright (C) 2021 Adrian Carpenter, et al
 #
 # This file is part of Pingnoo (https://github.com/nedrysoft/pingnoo)
 #
 # An open-source cross-platform traceroute analyser.
 #
-# Created by Adrian Carpenter on 27/03/2020.
-#
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
+#
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import platform
-import os
-import shutil
-import glob
-import tempfile
 import argparse
+import datetime
```

Comment:
PEP8 recommends putting these in alphabetical order. I would recommend checking out the free version of PyCharm as well; it helps do these things for you automagically.

Reply:
Cool.  I do actually do that in the main code, although where are segmentations basically:

<corresponding header>
<newline>
<private includes>
<newline>
<system includes>

Where each section is sorted alphabetically.

The deploy script hasn't seen as much love as it could have, it has grown organically as a method to easily create deployable images.  I appreciate you taking the time to make suggestions & supply PR's, it's a massive help.

I have a license of PyCharm, I'll have a look at your PR inside it.